docs/UserGuide.md

```diff
@@ -194,6 +193,55 @@ Format: `list`
 
 ![UiUList](images/UiList.png)
 
+---
+### Finding a student : `find`
+
+Finds students whose names contain any of the given keywords.
+
+Format: `find KEYWORD [MORE_KEYWORDS]`
+
+- The search is case-insensitive. e.g. `alex` will match `Alex`
+- The order of the keywords does not matter. e.g. `Yeoh Alex` will match `Alex Yeoh`
+- Only the name is searched.
+- Only full words will be matched e.g. `Han` will not match `Hans`
+- Persons matching at least one keyword will be returned. e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+
+Example:
+
+`find tan` returns `Tan Xiao Ming` and `John Tan`
+
+![UiFind](images/UiFind.png)
+
+---
+### Deleting a student: 'delete'
+
+Deletes the specified person from the student list.
+
+Format: `delete INDEX`
+
+- Deletes the person at the specified `INDEX`
+- The index refers to the index number shown in the Student's Details panel (bottom left)
+- The index must be a positive integer. e.g. `1, 2, 3, ...`
+
+Examples:
+- `list` followed by `delete 2` deletes the 2nd person in the Student's Details panel.
+- `find Betsy` followed by `delete 1` deletes the 1st person in the Stundent's Details panel.
```

Comment:
For which lines do we include the full-stop? Maybe you can explain the rationale in the description of the PR.